database/migrations/2017_03_13_212917_CreateSharedExpenseTable.php

```diff
@@ -18,6 +18,7 @@ public function up()
             $table->float('amount_owed');
             $table->string('secondary_username');
             $table->string('comments');
+            $table->timestamp('date_added');
```

Comment:
The date_settled is in the Settled Expenses table. Talked to Vince and he said he prefers to have the Settled Expenses table